### PR TITLE
Replace incoming-connections with multiserver fix

### PR DESCRIPTION
**Pull request by christianbundy**
This removes the incoming-connections workaround in favor of the correct
fix from the Multiserver module. Since that module's fix hasn't been
merged yet, this PR uses patch-package to manage the patch and ensure
that we're using the correct configuration.

Pending https://github.com/ssbc/multiserver/pull/49

### CODE REVIEW

**Comment by christianbundy** [comment]
Nope, that diff just adds a patch file to the directory. :man_facepalming:

File: defaults.js [hunk 1]
```diff
@@ -46,7 +47,20 @@ module.exports = function setDefaults (name, config) {
   if (!config.connections.incoming) {
     // if no incoming connections have been set,
     // populate this with some rad-comprehensive defaults!
-    config.connections.incoming = incomingConnections(config)
+    config.connections.incoming = {
+      net: [{
+	host: config.host || '::',
```
Comment (dominictarr) [inline defaults.js]: this should have one more indent inside the `net: [{...}]` object

**Comment by dominictarr** [comment]
if you use `::` as the address how does it behave when you do `getAddress('public')` `getAddress('local')` `getAddress('device')`? does it give the correct results?

File: defaults.js (hunk 1 above)
Comment (christianbundy) [reply defaults.js]: Thanks. This was resolved in the open PR, #67.

**Pull request closed without merging**